blockly-core/core/ui/function_editor.js

```diff
@@ -825,7 +825,9 @@ Blockly.FunctionEditor.prototype.setupParametersToolbox_ = function () {
 };
 
 Blockly.FunctionEditor.prototype.addEditorFrame_ = function () {
-  var left = Blockly.hasCategories ?
+  // if Blockly is in readOnly mode, don't pad. Otherwise, pad left
+  // based on the size of either the Toolbox or the Flyout
+  var left = Blockly.readOnly ? 0 : Blockly.hasCategories ?
```

Comment:
Brent added a similar check here yesterday https://github.com/code-dot-org/code-dot-org/pull/4395/files — instead of checking for readOnly here, it might make sense to check for isFlyout, in which case we can also prob. drop the explanatory comment